text/0000-fabric-private-chaincode-lite.md

```diff
@@ -658,38 +663,43 @@ The project team currently participates in the new Confidential Computing Consor
-    - aes-gcm 128
-  - usage and key-lifetime: 
-    - encryption of response to client requests (key generation by client, one key per request)
-    - encryption of values in ledger's K/V store (key generation inside enclave, one per chaincode
+- Symmetric encryption:
+  - Algorithm: AES-GCM 128
+  - Usage and key-lifetime: 
+    - Encryption of response to client requests (key generation by client, one key per request)
+    - Encryption of values in ledger's K/V store (key generation inside enclave, one per chaincode
       <!-- and shared only among all _registered_ enclaves of the same chaincode -->
       ),
-    - sealing of enclave state (key derived from SGX fuse keys via `EGETKEY` instruction, one key per <chaincode, platform> combination)
-- public key encryption
-  - algorithm:
-    - RSA-OAEP 3072
-  - usage and key-lifetime:
+    - Sealing of enclave state (key derived from SGX fuse keys via `EGETKEY` instruction, one key per <chaincode, platform> combination)
+- Public key encryption:
+  - Algorithm: RSA-OAEP 3072
+  - Usage and key-lifetime:
     <!-- - key distribution (ene enclave encryption key per <chaincode, platform> combination, key generation inside enclave) -->
-    - encryption of client requests (key generation of chaincode encryption key inside enclave, one per chaincode)
-- public key signatures
-  - algorithm:
-    - ECDSA secp256k1
-  - usage and key-lifetime:
-    - signing of
+    - Encryption of client requests (key generation of chaincode encryption key inside enclave, one per chaincode)
+- Public key signatures:
+  - Algorithm: ECDSA secp256k1
+  - Usage and key-lifetime:
+    - Signing of
       registration messages,
       <!-- key distribution messages -->
       and enclave endorsement (one enclave signature keys per <chaincode, platform> combination, key generation inside enclave)
 
+Next we summarize the key material used by the FPC components in addition to standard Fabric key material.
+
+- `enclave_sk` - Enclave signature key, used for FPC Chaincode endorsements. This key is kept inside the FPC Chaincode Enclave.
+- `enclave_vk` - Enclave signature verification key, used for FPC Chaincode endorsement validation and as enclave identifier. This key is made available through the Enclave Registry among all participants.
+- `chaincode_ek` - Chaincode encryption key, used to encrypt transaction arguments. This key is made available through the Enclave Registry among all participants.
+- `chaincode_dk` - Chaincode decryption key, used to decrypt transaction arguments. This key is kept inside the FPC Chaincode Enclave.
+- `sek` - State encryption (symmetric) key, used to encrypt/decrypt and authenticate the contents of chaincode state stored on the ledger. This key is kept inside the FPC Chaincode Enclave.
+- `return_encryption_key` - Chaincode Response Encryption Key, generated and sent by the client to the FPC Chaincode Enclave for response encryption. A fresh key for every invocation is used.
+
```

Comment:
Hmm, these keys are all mentioned in the section above including purpose and where it is created (hence kept). The only difference here is that you add the names. Seems a bit strange to duplicate everything? If you really think we have to name the keys as in the protobufs, why not just add them to the earlier section?